### ISSUE TITLE
Terraform tries to delete DATABASE_URL env var.

### ISSUE THREAD

**Issue by michal-januszkiewicz** [terraform-providers/terraform-provider-heroku#30]
Hi,

I have a config with a database addon which generated `DATABASE_URL` after the first time I ran `terraform apply` and everything was ok, the `DATABASE_URL` was generated and added to heroku app but then I noticed I need to change some of the env vars so I edited the terraform file and after running `terraform plan` the output showed me it would change `DATABASE_URL` to an empty string.

Here is what I have in config vars
```hcl
  config_vars {
    HANAMI_ENV = "production",
    NODE_MODULES_CACHE = "false",
    NPM_CONFIG_PRODUCTION = "false",
  }
```
As you can see there is no `DATABASE_URL` specified since it is computed.

And here is the output of `terraform plan`
```
~ heroku_app.staging
      config_vars.0.%:                     "4" => "3"
      config_vars.0.DATABASE_URL:          "postgres://agextps..." => ""
      config_vars.0.NODE_MODULES_CACHE:    "0" => "false"
      config_vars.0.NPM_CONFIG_PRODUCTION: "0" => "false"
```

**Comment by justincampbell** [comment]
Duplicate of #26, and being addressed in #25. In the meantime, you can pin the Heroku Terraform module to v0.1.0:

```hcl
provider "heroku" {
  version = "0.1.0"
}
```

Thanks! Sorry for the inconvenience.

**Comment by justincampbell** [comment]
[v0.1.2 has been released](https://github.com/terraform-providers/terraform-provider-heroku/blob/master/CHANGELOG.md#012-january-04-2018) with a fix for this. Please test and report any issues. Thanks!
